src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -343,18 +435,18 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
                         <div>    <span>- cml comment create report.md</span></div>
                       </Tooltip>
                     </Code>
-
+        
                     <ExampleBox title="CML Report">
-                      <a target="_blank" rel="noreferrer" href="https://gitlab.com/iterative.ai/cml-runner-example/-/merge_requests/2">
+                      <a target="_blank" rel="noreferrer" href="https://gitlab.com/iterative-test/cml-example-cloud-gpu/-/merge_requests/1">
                         <Image src="/img/gitlab/cml-runner-report.png" alt="GitLab Cloud report example" />
                       </a>
                     </ExampleBox>
-
+        
                   </Collapser>
                 )}
                 github={(
                   <Collapser>
-                    <Code filename=".github/workflows/cml.yaml" repo="https://github.com/iterative/cml-runner-base-case">
+                    <Code filename=".github/workflows/cml.yaml" repo="https://github.com/iterative-test/cml-example-cloud-gpu">
```

Comment:
again, non-GPU example.

```suggestion
                    <Code filename=".github/workflows/cml.yaml" repo="https://github.com/iterative/cml-runner-base-case">
```